frontend/getLabels.py

```diff
@@ -3,7 +3,6 @@
 labels=''
 rawLabels=json.loads(sys.argv[1])['labels']
 for rLabel in rawLabels:
-  if rLabel['name'].startswith("kiali-bot"):
```

Comment:
Do we even use `getLabels.py` anymore?  I thought we could just remove this. This technical debt may be from an old build system, but I'm not sure. I don't think it even makes sense just to remove this line, because in that case I'm not sure what the purpose of this script would then be.